htrace-core4-shaded/pom.xml

```diff
@@ -1,14 +1,18 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<!-- Licensed to the Apache Software Foundation (ASF) under one or more contributor
```

Comment:
From what I remember and according to https://www.apache.org/licenses/LICENSE-2.0.html we must retain the copyright/license of work that we redistribute. Here's that the shaded dependency. Any reason that you apply this change in the header here?